rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/10_synonyms_put.yml

```diff
@@ -3,6 +3,25 @@ setup:
       version: " - 8.8.99"
       reason: Introduced in 8.9.0
 
+---
+"Create synonyms with no ID creates ID automatically":
+  - do:
+      synonyms.put:
+        synonyms_set: test-update-synonyms
+        body:
+          synonyms_set:
+            - synonyms: "hello, hi"
+            - synonyms: "bye => goodbye"
+
+  - match: { result: "created" }
+
+  - do:
+      synonyms.get:
+        synonyms_set: test-update-synonyms
+
+  - length: { synonyms_set.0.id: 20 }
```

Comment:
Checking for length is the best I could think of to check the ID is included 🤷

Reply:
For this check we can use `is_true:`, for example: 

` - is_true: synonyms_set.0.id`

which means `synonyms_set.0.id` exists and is not `null` or `empty`

Reply:
TIL, thanks!